ee/codegen/src/generators/nodes/final-output-node.ts

```diff
@@ -15,9 +15,18 @@ export class FinalOutputNode extends BaseNode<
   protected DEFAULT_TRIGGER = "AWAIT_ANY";
   protected getNodeBaseGenericTypes(): AstNode[] {
     const stateType = this.getStateTypeOrBaseState();
-    const primitiveOutputType = getVellumVariablePrimitiveType(
-      this.nodeData.data.outputType
-    );
+
+    let primitiveOutputType: python.Type;
+    const outputWithSchema = this.nodeData.outputs?.find(output => output.schema);
```

Comment:
DevinAI this should be `this.nodeData.outputs?.find(output => output.key === 'value')`